src/components/VideoAsset.vue

```diff
@@ -15,28 +15,38 @@
     :should-wrap="!!deviceFrame"
     :device="deviceFrame"
   >
-    <video
-      ref="video"
-      :controls="showsControls"
-      :data-orientation="orientation"
-      :autoplay="autoplays"
-      :poster="normalisedPosterPath"
-      :muted="muted"
-      :width="optimalWidth"
-      playsinline
-      @loadedmetadata="setOrientation"
-      @playing="$emit('playing')"
-      @pause="$emit('pause')"
-      @ended="$emit('ended')"
-    >
-      <!--
-        Many browsers do not support the `media` attribute for `<source>` tags
-        within a video specifically, so this implementation for dark theme assets
-        is handled with JavaScript media query listeners unlike the `<source>`
-        based implementation being used for image assets.
-      -->
-      <source :src="normalizePath(videoAttributes.url)">
-    </video>
+    <div>
+      <span
+        v-if="alt"
+        :id="id || null"
+        class="visuallyhidden"
+      >
+        {{ alt }}
+      </span>
+      <video
+        ref="video"
+        :controls="showsControls"
+        :data-orientation="orientation"
+        :autoplay="autoplays"
+        :poster="normalisedPosterPath"
+        :muted="muted"
+        :width="optimalWidth"
+        :aria-describedby="id || null"
```

Comment:
Same comment from above: I think `:aria-describedby="id || null"` would be the same as `:aria-describedby="id"` since Vue will not actually populate the attribute if it is a falsy value.